runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -241,7 +241,7 @@ native2InterpJavaUpcallImpl(J9UpcallMetaData *data, void *argsListPointer)
 
 	VM_VMAccess::inlineEnterVMFromJNI(currentThread);
 	returnType = getReturnTypeFromMetaData(data);
-	returnsObject = ((J9NtcPointer == returnType) || (J9NtcStruct == returnType)) ? true : false;
+	returnsObject = ((J9NtcPointer == returnType) || (J9NtcStruct == returnType));
```

Comment:
The outermost parentheses are redundant.

Reply:
Fixed.